tests/test_queue.py

```diff
@@ -309,7 +309,7 @@ def test_enqueue_requeue_limit_invalid(self):
     def test_enqueue_cannot_serialize_payload(self):
         self.assertRaisesRegexp(
             BadArgumentException,
-            r'can not serialize.',
+            r'can\'t serialize',
```

Comment:
I have changed this last time as the `make test` was failing in my environment, it seems like I haven't used the correct version of msgpack-python, anyways if all tests are passed then its fine.